src/packages/StorageFileApi.ts

```diff
@@ -543,4 +624,21 @@ export default class StorageFileApi {
   private _removeEmptyFolders(path: string) {
     return path.replace(/^\/|\/$/g, '').replace(/\/+/g, '/')
   }
+
+  private transformOptsToQueryString(transform: TransformOptions) {
+    const params = []
+    if (transform.width) {
+      params.push(`width=${transform.width}`)
+    }
+
+    if (transform.height) {
+      params.push(`height=${transform.height}`)
+    }
+
+    if (transform.resize) {
+      params.push(`resize=${transform.resize}`)
+    }
+
+    return params.join('&')
+  }
```

Comment:
You might like to use the [search params API](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) here:

```javascript
const searchParams = new URLSearchParams()

if (transform.width) {
  searchParams.append('width', transform.width)
}

if (transform.height) {
  searchParams.append('height', transform.height)
}

if (transform.resize) {
  searchParams.append('resize', transform.resize)
}

return searchParams.toString()
```

Reply:
is there an isomorphic version for node too?

Reply:
Yep since Node 10 I believe! https://nodejs.org/api/url.html#class-urlsearchparams